src/main/java/com/sampleweb/pages/CarAndMotorbikePage.java

```diff
@@ -0,0 +1,101 @@
+package com.sampleweb.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+// A wild lenny appears ( ͡° ͜ʖ ͡°)
+public class CarAndMotorbikePage extends HomePage {
+
+    public static String PATH = "https://www.amazon.co.uk/"; // Amazon home page url
+    public String title = "Car & Motorbike";
+    public String pageTitle = "//h1[text()=\"Car & Motorbike\"]";
+    public String primeLabel = "//*[@aria-label='Amazon Prime']";
+
+    // TODO: Change file paths from absolute to relative
+    public String navTitle = "//*[@id=\"nav-subnav\"]/a[1]/span";
```

Comment:
Having hard coded indexes for the list of elements makes the code brittle and might break in the future if eg order changes.

Reply:
converted them to relative paths